recipe/meta.yaml

```diff
@@ -1,21 +1,22 @@
-{% set version = "0.9.0" %}
-{% set commit = "f9f8320339692d4134d1ef42c32cb7c8d547593e" %}
-{% set checksum = "6fa549d8aa01e457ae6b9962179caec3df9153659d5b3ff82449719d48517dc9" %}
+{% set version = "0.10.0" %}
+{% set commit = "07f142df646c9070fc9c627004dfdc96914fe487" %}
+{% set checksum = "943207a2fcc7ba8de0e50bdb6c6ea4e9ed7f7e7bf55f6b426d7f867f559e842d" %}
 
 package:
   name: arrow-cpp
   version: {{ version }}
 
-
 source:
   fn: {{ commit }}.tar.gz
   url: https://github.com/apache/arrow/archive/{{ commit }}.tar.gz
```

Comment:
We _can_ but people get grouchy when using dist.apache.org for software distribution instead of the mirror system. The problem with the mirror system is that it doesn't work properly with wget/curl

Reply:
Got it, using github and commit sha then